src/models/deployers/UniswapWrapperDeployer.js

```diff
@@ -0,0 +1,55 @@
+const BaseDeployer = require('./BaseDeployer')
+const CallsEncoder = require('../CallsEncoder')
+const logger = require('../../helpers/logger')('UniswapWrapperDeployer')
+
+const VERSION = 'v1.0'
+
+const VERIFICATION_HEADERS = [
+  'Commit sha: e96cb7fa7834f8727102a55f0ee1192387da0a26',
```

Comment:
We should merge and tag this repo first and then use here the tag commit sha

Reply:
Yes, I'll do that once it's merged, the problem is that I deployed it on Rinkeby before.